### DISCUSSION TITLE
How to donate to Laragon?

### DISCUSSION THREAD

**Discussion by hj-collab** [leokhoa/laragon#383] [General]
Question is in the title. Let me know. Thank you for making such a great product!

**Comment by Pen-y-Fan** [comment]
There are now two ways to donate to Laragon:

- <https://laragon.org/donate/>
- <https://github.com/sponsors/leokhoa?o=sd&sc=github>

Edit: Add Github sponsor link

**Reply by Sheldon-Monk** [reply]
Just an FYI:  I donated, and when I checked to make sure it went through, it appears the payment went to bitmana.io.  Issue is, when I look for https://bitmana.io, it presents with your phpmyadmin.   "????"  Seems strange to me, thought you should know.

Also, why Pen-y-Fan, and not Leo Khoa?  When making online payments, it's a bit scary to end up with responses from a name not seen before.

**Reply by Sheldon-Monk** [reply]
Okay, everyone else:  If you haven't donated yet, quit being so cheap and give the guy a few bucks.  Think about how much time and effort he's saved us all.

**Reply by Alain12345-1** [reply]
Looking for a place to donate.  Not obvious.

**Reply by germain-italic** [reply]
@Alain12345-1 litteraly the first (and only) link in the message you are replying to!

**Reply by Pen-y-Fan** [reply]
I've edited my original post and added the GitHub sponsor link too.

**Reply by Alain12345-1** [reply]
Monthly donnation activated.  Follow me to support such an awesome product.  I wish Laragon well so I talk with my limited wallet.

**Comment by leokhoa** [comment]
Thank you for supporting the development of Laragon.
No donation is too small, thanks <3

**Reply by Alain12345-1** [reply]
Looking for where to donate.

**Comment by tomerof** [comment]
This tool is excellent! love it!
I see that you stopped using the self hosted forum, if you need i can host it for you, for free.

**Comment by rescenic** [comment]
[@rescenic](https://github.com/rescenic?metadata_campaign=netlifysendosslove2022) sent you a card to say “Thank you!”
https://oss.love/card/6Gl1QmAmN

**Comment by RAMINCNZ** [comment]
When I can I will..

I'm so Sorry you have had some difficulties.. I do know the feeling and will probably have to give up my online presence very soon for exactly the same reasons..

Best Wishes Going Forward.

**Comment by kresnasatya** [comment]
Hi @leokhoa, do you have plan to open [GitHub sponsor page](https://github.com/sponsors)? Many developers around the world open GitHub sponsor as a part of appreciation in open source software. IMO, Your software (Laragon) is a great software that should be appreciated by other developers (include myself). So, I would like to appreciate your work in GitHub sponsor. I understand that you have open donation in PayPal. If you don't mind, I hope you open GitHub sponsor page as another gate open donation to appreciate your work.

**Reply by leokhoa** [reply]
@kresnasatya: My GitHub Sponsors profile is live!
https://github.com/sponsors/leokhoa?o=sd&sc=github

**Reply by kresnasatya** [reply]
> @kresnasatya: My GitHub Sponsors profile is live!
> https://github.com/sponsors/leokhoa?o=sd&sc=github

Thanks @leokhoa, I have give you a sponsor. Hope it helps.

**Comment by leokhoa** [comment]
Thanks @kresnasatya - will do.

**Comment by tienloc1** [comment]
Just sent a meal to you @leokhoa, I appreciate your work!

**Reply by leokhoa** [reply]
Thank you for your meal @tienloc1!

**Comment by Sennik** [comment]
I do not use Laragon everyday but for sure once in a while for some years now. Hate to see you are struggling with it financially, so I just made a donation.

**Reply by leokhoa** [reply]
I have received your gift. Thank you!

**Comment by bau720123** [comment]
also sent a meal to @leokhoa 
please still work hard every day
very appreciate

**Reply by leokhoa** [reply]
Got it. Thank you [包四維]!

**Comment by hagege** [comment]
Thanks for Laragon. I just made a donation.
It's great and I'm very happy to see you keep working on it.

**Comment by Pen-y-Fan** [comment]
I'm happy to give my time and a donation to help Laragon be even better, keep it up @leokhoa!

**Reply by leokhoa** [reply]
It is much appreciated @Pen-y-Fan 
Your time is valuable. I am so thankful and proud of having you in Laragon community. Thank you!

**Comment by awaisshoukat45** [comment]
How we add another php version in laragon. mean  i want add 7.4.20 version of php

**Reply by hagege** [reply]
When using Laragon, it is very easy to install another PHP version. All you need to do is download the appropriate package from the following page. Let's take PHP 7.4 (https://windows.php.net/downloads/releases/php-7.4.13-Win32-vc15-x64.zip) as an example. In this case we download the version to the local computer. Then we unpack the PHP version in the following folder: C:\laragon\bin\php\php-7.4.8-Win32-vc15-x64 .
I have described this in more detail in the following German-language post: 
https://haurand.com/laragon-lokaler-server-wordpress-und-php-8/

**Comment by germain-italic** [comment]
I'm happy to give a donation too to @leokhoa ; every time I open Laragon I appreciate the time and complexity I save by the efforts you put in it.

Also, I think GitHub is a better place than the old forum for the community, but a static archive of the old docs pages (hosted on GH) would be useful (or even better: GH wiki conversion). I could help on my spare time. Let us know what you need!

**Comment by codtechzofficial** [comment]
First, I would like to thank you, @leokhoa, for the effort and time you spent creating Laragon. I would like to help you, but I don't know how to get in touch with you. Please reach me at rhuziel.doc@gmail(.)com

**Comment by bsaqqa** [comment]
You got new coffee bro 😅😉.
Keep going 💪💪
Thanks for support us with Laragon ❤️

**Comment by joleenshook** [comment]
Fellow users, please spend a few bucks a month sponsor this dude if you can. Great product.

**Comment by leokhoa** [comment]
@joleenshook 
I have seen your Github monthly sponsor. I was a little bit unhappy because the downloads of Laragon increase but the sponsors amounts decrease. Then, your action makes my day.
Thanks.

**Comment by germain-italic** [comment]
If you want proper support from companies, you must set up a foundation or your own company to make tax invoices.

I've given money from my own pockets, but the usage for Laragon is for a business (which is not mine) therefore if the owners cannot buy a proper license with tax invoice, they cannot send money for regulatory reasons.

**Comment by JuanPreuyt** [comment]
Made a small donation. 

Thanks for the great product, probably took a ton of work to put together and maintain, used it in conjunction with Xammp but it is better, lighter faster, and way more flexible. 

I hope more people sponsor.

**Comment by iamrobert** [comment]
@leokhoa

I love your software and wish more people donate. 

However, it's not obvious how to on the Laragon website. Can you put a button?

![](https://i.imgur.com/KBgBNlK.png)

**Reply by Sheldon-Monk** [reply]
We do, many of us, still no further development in months.  The site is down, and it's all been switched to github which comparatively is far more laborsome.

**Reply by leokhoa** [reply]
@iamrobert: Thanks for you kind wish. I will update the website.

@Sheldon-Monk: I have some plans for Laragon in 2024 when I have more free time. So, stay tune...

**Comment by dven88** [comment]
I made a very small donations via paypal a week or so ago but... was there someone to receive it at all? ^_^'
Especially because the payment went to bitmana.io and there is no way to tell it's related to laragon

**Comment by leokhoa** [comment]
@dven88 
I got your donation. Thanks Dario Venneri.
F.Y.I. bitmana.io is my side project.

**Reply by dven88** [reply]
nice, and thank you for creating Laragon :) really a great tool

**Comment by pizzucu** [comment]
I really love laragon. I hope you will continue your great work and I hope my little donation keeps you motivated :)

**Comment by fifisave** [comment]
LOVE LOVE LOVE !!! Thanks so much for this new version .. have a coffee for me !

**Comment by Zatovo** [comment]
How we add another node version in laragon. mean i want add Node 17

**Reply by dven88** [reply]
download the standalone binary of v17 from node's website, then go to the laragon bin/node folder and drop the unzipped folder there. Restart laragon and it will appear among the options
